tests/kernel/profiling/profiling_api/testcase.yaml

```diff
@@ -3,3 +3,8 @@ tests:
     arch_exclude: nios2 riscv32
     platform_exclude: em_starterkit
     tags: kernel
+  kernel.profiling.initstack:
```

Comment:
this is not a test, enabling and disabling configs does not test anything, we do not know if the code executed correctly, so achieving coverage by just touching the code is not enough, we need a proper test for the option that verifies it works.